curvefs/src/client/fuse_client.cpp

```diff
@@ -1471,5 +1548,32 @@ FuseClient::SetMountStatus(const struct MountOption *mountOption) {
     return CURVEFS_ERROR::OK;
 }
 
+void FuseClient::InitQosParam() {
+    ReadWriteThrottleParams params;
+    params.iopsWrite = ThrottleParams(FLAGS_fuseClientAvgWriteIops,
+                                      FLAGS_fuseClientBurstWriteIops,
+                                      FLAGS_fuseClientBurstWriteIopsSecs);
+
+    params.bpsWrite = ThrottleParams(FLAGS_fuseClientAvgWriteBytes,
+                                     FLAGS_fuseClientBurstWriteBytes,
+                                     FLAGS_fuseClientBurstWriteBytesSecs);
+
+    params.iopsRead = ThrottleParams(FLAGS_fuseClientAvgReadIops,
+                                     FLAGS_fuseClientBurstReadIops,
+                                     FLAGS_fuseClientBurstReadIopsSecs);
+
+    params.bpsRead = ThrottleParams(FLAGS_fuseClientAvgReadBytes,
+                                    FLAGS_fuseClientBurstReadBytes,
+                                    FLAGS_fuseClientBurstReadBytesSecs);
+
+    throttle_.UpdateThrottleParams(params);
+
+    int ret = bthread_timer_add(&throttleTimer_, butil::seconds_from_now(1),
```

Comment:
not  periodic tasks? maybe can reference [timer in curve](https://github.com/opencurve/curve/blob/master/curvefs/src/metaserver/heartbeat.h#L147)

Reply:
This is a timer task that refreshes variables once per second

Reply:
> This is a timer task that refreshes variables once per second
yes
![6d2cc34d-2b00-4cb5-a46a-f2804b95ccff](https://github.com/opencurve/curve/assets/13897133/9b1cd3c7-b114-4a84-a054-d874e40e7ab1)